packages/flutter/lib/src/widgets/snapshot_widget.dart

```diff
@@ -368,7 +377,7 @@ abstract class SnapshotPainter extends ChangeNotifier  {
   /// }
   /// ```
   /// {@end-tool}
-  void paintSnapshot(PaintingContext context, Offset offset, Size size, ui.Image image, double pixelRatio);
+  void paintSnapshot(PaintingContext context, Offset offset, Size size, ui.Image image, Size imageSize, double pixelRatio);
```

Comment:
Nit on the naming, its not really the image size. Maybe call it the srcSize? Not sure exactly.

Reply:
done